roxie/ccd/ccdmain.cpp

```diff
@@ -739,7 +739,7 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                     // Makes debugging easier...
                     IPropertyTree *service = topology->addPropTree("services");
                     service->setProp("@name", "query");
-                    service->setPropInt("@port", 9876);
+                    service->setPropInt("@servicePort", 9876);
```

Comment:
is this correct? Shouldn't it be listening to @port (the internally defined port to use), which should default to @servicePort, if @port was not defined in values yaml

Reply:
or maybe this isn't being used to listen to here,
but is being used contact the exposed service ?

Reply:
Roxie uses a single port to specify both internal and external - they are not allowed to differ.